lib/chat_web/live/main_live/layout/message_input.ex

```diff
@@ -25,11 +31,7 @@ defmodule ChatWeb.MainLive.Layout.MessageInput do
           id="dialog-form"
           class="basis-[99%] flex items-center justify-between"
           phx-change={JS.dispatch("chat:set-input-size", to: "#dialog-input")}
-          phx-submit={
-            JS.push("dialog/text-message")
-            |> JS.dispatch("chat:clear-value", to: "#dialog-input")
-            |> JS.dispatch("chat:set-input-size", to: "#dialog-input")
-          }
+          phx-submit={submit_dialog_text_message(%{writable: @writable})}
```

Comment:
It would be great to have some color indication as well

Reply:
I was about text-inputs and buttons
Popup was good enough. Now it is too scary. Text is following

### Read only mode
The device switched into read only mode. Storage drive needs to be upgraded.

Reply:
<img width="1096" alt="Screenshot 2023-02-02 at 21 46 52" src="https://user-images.githubusercontent.com/90610934/216434275-5ebacce5-a042-47db-a4a0-780589c5a0a4.png">